provider/datagen/src/lib.rs

```diff
@@ -96,6 +103,7 @@ use std::path::{Path, PathBuf};
 /// [`DataProvider`] backed by [`SourceData`]
 #[allow(clippy::exhaustive_structs)] // any information will be added to SourceData
 #[derive(Debug, Clone)]
+#[doc(hidden)]
```

Comment:
why?

Reply:
This is an implementation detail of our datagen. It's public because `verify-zero-copy` needs it, but I don't think it should be public-public

Reply:
But it's not an implementation detail. It's what people can use if they want to read directly from CLDR/ICUExport source data at runtime.

Reply:
ok